pulpcore/cli/common/context.py

```diff
@@ -724,6 +725,20 @@ class PulpRemoteContext(PulpEntityContext):
     }
 
 
+class PulpAcsContext(PulpEntityContext):
+    ENTITY = _("ACS")
+    ENTITIES = _("ACSes")
+    HREF_PATTERN = r"acs/(?P<plugin>[\w\-_]+)/(?P<resource_type>[\w\-_]+)/"
+    ID_PREFIX = "acs"
```

Comment:
Sorry for the silent merge conflict here. This class is (by this time) already defined at the end of the file.